_grab_ui.py

```diff
@@ -609,12 +619,19 @@ def _select(self, direction: DirectionStr,
                 self.mark, old_point, self.point))
 
     def move(self, direction: DirectionStr) -> None:
-        self._select(direction, NoRegion)
+        if self.mode == 'normal':
+            self._select(direction, NoRegion)
+        else:
+            self._select(direction, self.mode_types[self.mode])
```

Comment:
Since `mode_types['normal']` is `NoRegion`, this conditional seems to be unnecessary. Just use the statement in the `else` branch in all cases.